diff --git a/node_modules/@10play/tentap-editor/android/build.gradle b/node_modules/@10play/tentap-editor/android/build.gradle
index 3aec145..bf89722 100644
--- a/node_modules/@10play/tentap-editor/android/build.gradle
+++ b/node_modules/@10play/tentap-editor/android/build.gradle
@@ -15,7 +15,6 @@ buildscript {
 }
 
 apply plugin: "com.android.library"
-apply plugin: "kotlin-android"
 
 def getExtOrDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : project.properties["TenTap_" + name]
@@ -45,11 +44,11 @@ android {
     }
   }
 
-  compileSdkVersion getExtOrIntegerDefault("compileSdkVersion")
+  compileSdk 36
 
   defaultConfig {
     minSdkVersion getExtOrIntegerDefault("minSdkVersion")
-    targetSdkVersion getExtOrIntegerDefault("targetSdkVersion")
+    targetSdkVersion 36
   }
 
   buildTypes {
